ssl/statem/statem_clnt.c

```diff
@@ -4027,7 +4027,9 @@ int ssl_cipher_list_to_bytes(SSL_CONNECTION *s, STACK_OF(SSL_CIPHER) *sk,
 {
     int i;
     size_t totlen = 0, len, maxlen, maxverok = 0;
-    int empty_reneg_info_scsv = !s->renegotiate;
+    int empty_reneg_info_scsv = !s->renegotiate
+                                && (SSL_CONNECTION_IS_DTLS(s)
```

Comment:
Any plans to support DTLSv1.3 (RFC9147)? The RFC is a year old. Could this be future-proofed?

Reply:
Adding DTLSv1.3 support is a major update and there are *lots* of existing places which will need to be updated to handle this. I don't think one more makes a difference.